### ISSUE TITLE
Up for group order ?

### ISSUE THREAD

**Issue by kds69** [formtapez/ZigUP#25]
Dears,
Since I am not competent to solder SMTs I have take a quote for 5x ZigUp assembled by supplier.
The ticket price is $80 by item +shipping cost from France.
Just let me know if you are interested.

**Comment by codingcatgirl** [comment]
Soldering the SMD components is pretty easy. I'm by no means very experienced at soldering but with a few online tutorials it was fairly easy to do. It drops the price to <20$ per item.

**Comment by codingcatgirl** [comment]
Just to add: If you are willing to pay $80 (where are you even ordering? JLCPCB has to be cheaper) you might as well order a comemrcially available product similar to ZigUP for less money, giving you the added benefit of CE compliance etc.

**Comment by gsieben** [comment]
This is far from an acceptable and fair price. The price is usurious.

**Comment by kds69** [comment]
> Soldering the SMD components is pretty easy. I'm by no means very experienced at soldering but with a few online tutorials it was fairly easy to do. It drops the price to <20$ per item.

Really ? Will you assemble and solder it for me please for less than $20?
Just the one of the component is already $14... glad to read rest is $6 including PCB.

**Comment by kds69** [comment]
« Where are you even ordering »? Please pay a bit of respect in your reply.
So you did yourself have requested for a real quotation somewhere? Curious to read from it.

Now, if you know a  commercial item that have even half of all the features this one has, then please share with the community, would be fatastic, franky speaking.

**Comment by kds69** [comment]
> This is far from an acceptable and fair price. The price is usurious.

I am seriously extremely disappointed by the aggressivity of the reaction of people here.
Unbelievebale !
For sure none of you have spent a minute to look into this project and figure out what budget it is for the components, PCB and then man hours to assemble.
The rapidity of the replies tells a lot anyway...

**Comment by codingcatgirl** [comment]
Yes. I'm sure noone in here knows what this repository is about, we are all just random strangers who decided to comment on your post. Absurd accusation.

Anyway, since you indirectly asked for it, here is actually the result of me, weeks ago, spending much more than a minute "to look into this project and figure out what budget it is for the components, PCB". In fact, afterwards, i assembled several ones of them for me and two friends of mine. 

Still don't believe me? Here's the octopart BOM i created in the process; https://octopart.com/bom-tool/wTtv413B As you'll see, it indeed adds up to <20$ total per unit, including PCB. (I am curious what component you think costs $14 on it's own already because that's definitely not the experience i've made.)

As for the "and then man hours to assemble.". With a bit of work, you should be able to use JLCPCB's SMT assembly service. They do one SMT assembly order per month per customer for free even, so you only pay for the parts (and small extra costs for parts that aren't in their default parts library). Yes, they will only assemble one side for you, but that will cover a lot. You should be able to get the top side mostly assembled, with the Power Supply and the Relais likely to be missing, but those are Through-Hole Components, so they are easy to solver even if you don't want to touch SMD.

As for the commercial product: Will you really be using all the features at once? All hidden under your light switch? It sounds more likely that you will only be using some of them in each installation. That changes the scope a lot. The main feature of this is after all the mount below a light switch. There are countless products like that available and can easily be found on amazon for $30-50. If you to measure current consumption or control led strips, there are products available for this as well, definitely well below $80. 

Point is: Paying $80 for this is pretty absurd and i doubt anyone will be interested in that offer, given how much cheaper you can do it yourself. I hope i was able to help you understand how you should be able to cut your costs if you want to.

**Comment by kds69** [comment]
Thank you. This is a much constructive and respectful way to approach it.
I can imagine you are in shock with the price but that is what it is for having it assembled for 5 units only, and I didn’t want to engage in a twice+ bigger batch quantity. 

 Very interested to know if those prices are including import taxes and shipping cost?. I really look for total because I received tempting quotes which happens to be way more when you have everything counted.

One of the quickest, but not the cheapest for sure, I kept is PCBWay, for the full service.
Man hours is VERY expensive.

I appreciate the link to this company which I didn’t happen to know. I see this much better price anyway for particular item I mentioned to what I could find (outside sourced by myself to assemble, which also I didn’t want to engage to). Rest is comparable.
Now I am still curious to see what would be the real total price for all full assembly and delivered in less than 4 weeks only, as compared to 14 weeks (which is very acceptable though), including shipping cost and taxes. I will ceh k put and come back with a better offer if by chance it is.

In my case, I can’t afford spend several hours to solder those, nor well equiped, and eventually will ruin it and waste 1 or 2, and not do it in secure manner for use. I’m sure I am not the only one by far. You have this competence, I don’t.

Why didn’t you do what you recommend about commerical items ? I don’t get the point. Do you mean ZipUp commercial item or Zigbee whatever relays, binary sensor etc ? I couldn’t find any commerical version of ZigUp but agin glad to know any.
Indeed, I couldn’t find a commercial item that can do all features I need, taken individually all will be much more expensive, tidious to set-up and maintain and overall too voluminous anyway for my projects.

**Comment by kds69** [comment]
Ok it seems JLPCB doesn't provide full assembly services and to get to the low prices , one has to deal with the coupon system. That's the trick. (Still the insourced components are very well priced indeed).
With 10 pieces the PCBWay quote will drop very rapidly apparently, but this is chicken and egg. I can't commit on 10 myself. Still investigating.

**Comment by codingcatgirl** [comment]
>Thank you. This is a much constructive and respectful way to approach it. 
My first answer was in no way less respectful.

> Why didn’t you do what you recommend about commercial items?
Because ZigUP is cheaper and i like building things myself.

> Do you mean ZipUp commercial item or Zigbee whatever relays, binary sensor etc ?
The latter.

> Ok it seems JLPCB doesn't provide full assembly services and to get to the low prices , one has to deal with the coupon system. That's the trick. (Still the insourced components are very well priced indeed).
They do provide full assembly service for one side of any PCB you make with them. I'm really confused what you mean with "to get the low prices"… i mean after all the base fee is otherwise just $7. Still pretty low. Also, their coupon system usually applies coupons automatically, there isn't really anything you need to do, so there really is no "trick". Delivery is fairly cheap, takes usually about a week to Europe and they take care of customs so you don't need to go through that trouble.

But, yes, if you really want it fully assembled and just put enough money on the counter to have to do no work yourself whatsoever, then it's probably going to be more expensive. I'm afraid to say that this probably doesn't put you at the typical target audience of a DIY hardware project, so you will likely have a hard time finding other people interested in spending that kind of money. Mind you though that since you will need to flash the ZigBee Chip, you will still have to do some work that can be considered hardware work.

**Comment by kds69** [comment]
Thanks for the additional information, really.

For occasional DIY like me, even $100 would be a bargain provided the functionality offered by this repository, with a procudred ready-to-go solution, no brainer, no risk (as compared to me soldering and investing in good soldering station).
"where are you even..", "not fair", "usurious"... were probably not the most welcoming and respectful wordings, were they?
Not everybody is looking for the absolute cheapest ever and willing to try electronics DIY, this is what this is about in my case.

I am not an expert of demand-offer in wide spectrum of DIY audience, so I started with what I could come with: 5 items max for max 5 people.
I just posted here, naively, a first proposal to see interest or reactions for people like me.
I'll try to get a better price anyway.

Glad you share your experience and thoughts, this is highly appreciated and I will then have a second look.
